packages/headless/src/controllers/facets/range-facet/date-facet/date-range.ts

```diff
@@ -71,7 +79,7 @@ export function buildDateRange(config: DateRangeOptions): DateRangeRequest {
   };
 }
 
-function buildDate(rawDate: DateRangeInput, options: DateRangeOptions) {
+function buildDate(rawDate: string | number | Date, options: DateRangeOptions) {
```

Comment:
If we store this type as `AbsoluteDate`, then `DateRangeInput` could become `AbsoluteDate | RelativeDate`